### PR TITLE
fix #15 use simplified api

### LINKED ISSUE #15: API change proposal
Instead of
```js
this.get('scroller.scrollToElementId').perform(...arguments);
```
I would suggest to use
```js
this.get('scroller').scrollToElementId(...arguments);
```
and have that method `perform()` and return the task 🤔 

/cc @trabus

### PR DESCRIPTION
**Pull request by trabus**

### CODE REVIEW

**Comment by trabus** [comment]
@Turbo87 will wait for your feedback before merging.

File: addon/services/scroller.js [hunk 1]
```diff
@@ -32,14 +32,19 @@ export default Service.extend({
...
+  scrollToElementId(elementId, options = {}) {
     let element = document && document.getElementById(elementId);
     assert(`An element with the id: '${elementId}' could not be found in the DOM. Be sure to check that it has been rendered before attempting to scroll to it.`, element)
-    yield this.get('scrollToElement').perform(element, options);
+    return this.scrollToElement(element, options);
+  },
+
+  // task version so you can use the perform helper
+  scrollToElementIdTask: task(function * () {
+    yield this.scrollToElementId(...arguments);
```
Comment (Turbo87) [inline addon/services/scroller.js]: I think I'd implement this in reverse. Have most of the code logic in the task and have the method do nothing more than perform the task.

```js
  // scroll to an element by id
  scrollToElementId(elementId, options = {}) {
    this.get('scrollToElementIdTask').perform(elementId, options);
  },

  // task version so you can use the perform helper
  scrollToElementIdTask: task(function* (elementId, options = {}) {
    let element = document && document.getElementById(elementId);
    assert(`An element with the id: '${elementId}' could not be found in the DOM. Be sure to check that it has been rendered before attempting to scroll to it.`, element)
    yield this.scrollToElement(element, options);
  }),
```

File: addon/services/scroller.js [hunk 2]
```diff
@@ -56,7 +61,12 @@ export default Service.extend({
...
-    yield this.get('scrollTo').perform(start, end, options);
+    return this.get('scrollTo').perform(start, end, options);
+  },
+
+  // task version so you can use the perform helper
+  scrollToElementTask: task(function * () {
+    yield this.scrollToElement(...arguments);
   }),
 
   // start position, end position, duration in ms, easetype
```
Comment (Turbo87) [inline addon/services/scroller.js]: I think for `scrollTo()` this pattern would make sense too 🤔

**Comment by trabus** [comment]
@Turbo87 last chance for input! (for this release)

**Pull request merged**